### ISSUE TITLE
Code is Not Formatted - All Sections

### ISSUE THREAD

**Issue by liamrahav** [MakeSchool-Tutorials/Peeved-Penguins-SpriteBuilder-Swift#6]
The code is formatted like this:

> func play() { let gameplayScene = CCBReader.loadAsScene("Gameplay")CCDirector.sharedDirector().presentScene(gameplayScene) }

instead of this:

```
func play() {
        let gameplayScene = CCBReader.loadAsScene("Gameplay")
        CCDirector.sharedDirector().presentScene(gameplayScene)
    }
```

It's pretty difficult to read, and a switch from quotes to code formatting would make the experience much better.

**Comment by dionlarson** [comment]
Yeah, its just not indented enough for it to get processed by the markdown editor. I thought I fixed this earlier but will do again/look into it.

**Comment by dionlarson** [comment]
Fixed in 31bcf72a62ba71f780c9232b135d311793762716
